omero_marshal/decode/decoders/plate.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Glencoe Software, Inc. All rights reserved.
+#
+# This software is distributed under the terms described by the LICENCE file
+# you can find at the root of the distribution bundle.
+# If the file is missing please request a copy by contacting
+# jason@glencoesoftware.com.
+#
+
+from ... import SCHEMA_VERSION
+from .annotation import AnnotatableDecoder
+from omero.model import PlateI
+from omero.rtypes import rstring
+
+
+class Plate201501Decoder(AnnotatableDecoder):
+
+    TYPE = 'http://www.openmicroscopy.org/Schemas/OME/2015-01#Plate'
+
+    OMERO_CLASS = PlateI
+
+    def decode(self, data):
+        v = super(Plate201501Decoder, self).decode(data)
+        self.set_property(v, 'name', data.get('Name'))
+        self.set_property(v, 'description', data.get('Description'))
+        self.set_property(v, 'columnNamingConvention',
+                          data.get('ColumnNamingConvention'))
+        self.set_property(v, 'rowNamingConvention',
+                          data.get('RowNamingConvention'))
+        self.set_property(v, 'columns', data.get('Columns'))
+        self.set_property(v, 'rows', data.get('Rows'))
+        # self.set_property(v, 'defaultSample', data.get('FieldIndex'))
+        v.setDefaultSample(rstring(data.get('FieldIndex')))
```

Comment:
The tests pass here using ```v.setDefaultSample()``` but when I tried to do 
``` self.set_property(v, 'defaultSample', data.get('FieldIndex'))```
I get test failing with:
```
tests/unit/test_container_decoders.py::TestScreenDecoder::test_screen_with_plates_decoder 
INTERNALERROR> Traceback (most recent call last):
INTERNALERROR>   File "/usr/local/lib/python2.7/site-packages/_pytest/main.py", line 96, in wrap_session
...
INTERNALERROR>   File "/Users/wmoore/Desktop/OMERO/openmicroscopy/dist/lib/python/omero/rtypes.py", line 533, in __getattr__
INTERNALERROR>     raise AttributeError(attr)
INTERNALERROR> AttributeError: _val
```
which I'm finding it hard to debug.
Any ideas @chris-allan @sbesson?
 Thanks.